website/www/site/content/en/documentation/ml/multi-model-pipelines.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Multi-model pipelines"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Multi-model pipelines
+
+Apache Beam allows you to develop multi-model pipelines. In this specific scenario, you can ingest
+and transform some input data, run it through a model, and then pass the outcome of your first model
+into a second model. This page explains how multi-model pipelines work and gives an overview of what
+you need to know to build one.
+
+Before reading this section, it is recommended that you become familiar with the information in
+the [Pipeline development lifecycle](https://beam.apache.org/documentation/pipelines/design-your-pipeline/)
+.
+
+## How to build a Multi-model pipeline with Beam
+
+A typical machine learning workflow involves a series of data transformation steps such as data
+ingestion, data processing tasks, inference, and post-processing. Beam enables you to orchestrate
+all of those steps together by encapsulating them in a single Beam DAG. This allows you to build
+resilient and scalable end-to-end machine learning systems.
+
+To deploy your machine learning model in a Beam pipeline, you can use
+the [`RunInferenceAPI`](https://beam.apache.org/documentation/sdks/python-machine-learning/) which
```

Comment:
Can we also change the documentation on this [page](https://github.com/apache/beam/blob/master/website/www/site/content/en/documentation/sdks/python-machine-learning.md#ensemble-pattern) to match the wording here? I think I found 2 instances of the word "ensemble" in the other page.